server/src/pouch-db-app.ts

```diff
@@ -54,10 +56,46 @@ class PouchDbApp extends AppBase {
    * https://github.com/pouchdb/pouchdb-server
    */
   private addPouchRoutes(): void {
+    // If VM lives at non-root prefix, works but fauxton not fully working yet
+    const urlPrefix = process.env[VM_URL_PREFIX] || '/';
+    const config = {
+      mode: 'fullCouchDB', inMemoryConfig: true,
+      "httpd": {
+        "enable_cors": true
+      },
+      "cors": {
+        "origins": "https://skarabrae.org",
+        "credentials": true,
+        "headers": "accept, authorization, content-type, origin, referer",
+        "methods": "GET, PUT, POST, HEAD, DELETE"
+      }
+    };
+
+
     if (process.env[ON_DISK_DB]) {
       const dboptions = {prefix: '/personalcloud/'} as PouchDB.Configuration.RemoteDatabaseConfiguration;
       const onDiskPouchDb = PouchDB.defaults(dboptions);
-      this.express.use('/', PouchDbServer(onDiskPouchDb, {mode: 'fullCouchDB', inMemoryConfig: true}));
+      const pouchDbRouter = PouchDbServer(onDiskPouchDb, config);
```

Comment:
Below seems ok to land PR but further cleanup appears warranted, perhaps file an issue to restructure in whatever manner is appropriate? It looks like there's both potentially more complex LB logic we might want eventually, and some kind of Fauxton issue. Possible both of these are fine as implemented and no follow up issue needed but figured to ask while reading.